src/RadioButtonGroup/RadioButtonGroup.svelte

```diff
@@ -39,9 +45,13 @@
 
   const dispatch = createEventDispatcher();
   const selectedValue = writable(selected);
+  const groupName = writable(name);
+  const groupRequired = writable(required);
 
   setContext("RadioButtonGroup", {
     selectedValue,
+    groupName: { subscribe: groupName.subscribe },
+    groupRequired: { subscribe: groupRequired.subscribe },
```

Comment:
What's the reason for only assigning the subscribe method?

Reply:
Essentially turns the store into a readable, so it cannot be modified by descendants using the context. There's a utility function that could be used to do that, too.